x/examples/website-wrapper-app/doctor

```diff
@@ -0,0 +1,206 @@
+        dependency_found=0
+    fi
+    if [[ $dependency_found -eq 1 ]]; then
+        return 0
+    else
+        return 1
+    fi
+}
+
+check_version_ge() {
+    local current_version="$1"
+    local required_version="$2"
+    if [[ "$(printf '%s\n' "$required_version" "$current_version" | sort -V | head -n1)" == "$required_version" ]]; then
+        return 0
+    else
+        return 1
+    fi
+}
+
+echo "--- Outline SDK Web Wrapper Doctor ---"
+echo "Checking common dependencies..."
+
+check_dependency "Node.js" "node" "node --version" "" "" "$URL_NODE"
+if check_dependency "Go" "go" "" "" "" "$URL_GO"; then
+    go_version_full=$(go version)
+    go_version_num=$(echo "$go_version_full" | awk '{print $3}' | sed 's/^go//')
+    printf "  Checking Go version (requires >= ${MIN_GO_VERSION})... "
+    if check_version_ge "$go_version_num" "$MIN_GO_VERSION"; then
+        printf "${GREEN}${CHECK_MARK} Version ${go_version_num} is sufficient.${NC}\n"
+    else
+        printf "${RED}${CROSS_MARK} Version ${go_version_num} is too low.${NC}\n"
+        printf "  Please upgrade Go to version ${MIN_GO_VERSION} or higher.\n"
+        printf "  Installation guide: ${BLUE}%s${NC}\n" "$URL_GO"
+        all_ok=0
+    fi
+fi
+
+echo ""
+echo "Checking OS..."
+printf "Checking for macOS (required for iOS builds)... "
+if [[ "$(uname)" == "Darwin" ]]; then
+    printf "${GREEN}${CHECK_MARK} Found (macOS)${NC}\n"
+    is_macos=1
+else
+    printf "${YELLOW}${WARNING_MARK} Not macOS ($(uname)). iOS builds require macOS.${NC}\n"
+fi
+
+echo ""
+echo "Checking iOS specific dependencies..."
+if [[ $is_macos -eq 1 ]]; then
+    check_dependency "Xcode Command Line Tools" "xcodebuild" "xcodebuild -version" "" "" "$URL_XCODE" "$HINT_XCODE_CLI"
+    check_dependency "Cocoapods" "pod" "pod --version" "" "" "$URL_COCOAPODS" "$HINT_COCOAPODS"
+else
+    printf "${YELLOW}Skipping iOS checks (not on macOS)${NC}\n"
+fi
+
+echo ""
+echo "Checking Android specific dependencies..."
+# Java check - specifically looking for 17
+check_dependency "Java JDK" "java" "java -version" "version \"17." "Outline SDK requires OpenJDK 17." "$URL_JAVA_17" "$HINT_JAVA_17" "stderr"
+check_dependency "Java Compiler" "javac" "javac -version" "javac 17." "Ensure JDK (not just JRE) is installed and in PATH." "$URL_JAVA_17" "$HINT_JAVA_17"
+
+
+# Android SDK check
+printf "Checking for Android SDK (ANDROID_HOME or ANDROID_SDK_ROOT)... "
+android_sdk_path=""
+if [[ -n "$ANDROID_HOME" ]] && [[ -d "$ANDROID_HOME" ]]; then
+    printf "${GREEN}${CHECK_MARK} Found (ANDROID_HOME=${ANDROID_HOME})${NC}\n"
+    android_sdk_path="$ANDROID_HOME"
+elif [[ -n "$ANDROID_SDK_ROOT" ]] && [[ -d "$ANDROID_SDK_ROOT" ]]; then
+    printf "${GREEN}${CHECK_MARK} Found (ANDROID_SDK_ROOT=${ANDROID_SDK_ROOT})${NC}\n"
+    android_sdk_path="$ANDROID_SDK_ROOT"
+else
+    printf "${RED}${CROSS_MARK} Not found.${NC}\n"
+    printf "  Please set either the ANDROID_HOME or ANDROID_SDK_ROOT environment variable\n"
```

Comment:
IIRC, one of the environment variables is deprecated, maybe we don't need to mention that one?